jina/parsers/peapods/pea.py

```diff
@@ -38,7 +38,7 @@ def mixin_pea_parser(parser):
     # hidden CLI used for internal only
 
     gp.add_argument('--pea-id', type=int, default=-1,
-                    help='the id of the storage of this pea, only effective when `separated_workspace=True`'
+                    help='the id of the storage of this pea`'
```

Comment:
u say, needed when multiple peas run in parallel inside a same Pod

Reply:
lets do the documentation in a separated ticket

Reply:
let's add it here one second